cmd/commands/account/command.go

```diff
@@ -245,12 +251,18 @@ func (c *command) register() {
 		return
 	}
 
-	c.registerIdentity(id.Address)
+	c.registerIdentity(id.Address, ctx.String(flagToken.Name))
 }
 
-func (c *command) registerIdentity(identity string) {
-	err := c.tequilapi.RegisterIdentity(identity, "", new(big.Int).SetInt64(0), nil)
+func (c *command) registerIdentity(identity, token string) {
+	var validToken *string
+	if token != "" {
+		validToken = &token
+	}
+
+	err := c.tequilapi.RegisterIdentity(identity, "", new(big.Int).SetInt64(0), validToken)
 	if err != nil {
+		fmt.Println(err)
```

Comment:
Should it be `clio.Error` for some color?

Reply:
This was left here because sometimes i forget to remove things after debugging. removed.